generators/utils.js

```diff
@@ -344,3 +370,457 @@ function buildEnumInfo(field, angularAppName, packageName) {
 function copyObjectProps(toObj, fromObj) {
     Object.assign(toObj, fromObj);
 }
+
+
+/**
+ * get a property by path in a array
+ *
+ * @param {array} array - array in which we want to search
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ * @return {string} the value of property or undefined
+ */
+function getPropertyInArray(array, name, generator) {
+    return _.get(array, name);
+}
+
+/**
+ * Update or create a property by path in a array
+ *
+ * @param {array} array - array in which we want to update
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ * @param {string} value - (optional) value of the property
+ */
+function updatePropertyInArray(array, name, generator, value) {
```

Comment:
Please use lodash directly. This method doesnt add any value so please remove it